gitlab/integration_test.go

```diff
@@ -800,12 +807,10 @@ var _ = Describe("GitLab Provider", func() {
 		repoName := fmt.Sprintf("test-user-repo-creation-%03d", rand.Intn(1000))
 		repoRef := newUserRepoRef(testBaseUrl, "yadda-yadda-yada", repoName)
 
-		repo, err := c.UserRepositories().Create(ctx, repoRef, gitprovider.RepositoryInfo{})
```

Comment:
we should change the test name here too.

Reply:
Okay